### ISSUE TITLE
Issue with tables in Backgrounds

### ISSUE THREAD

**Issue by namtrah** [ebullient/fc5-convert-cli#9]
Tables seem to have an issue. in some of the backgrounds. Bonds, Ideals and sometime Flaws are being munged with previous tables. Happens mostly when there are tables back to back n the XML. An example is the output of the acolyte background. Many others, but probably fixing one will fix most of the others.
Example:
```
|d8 | Personality Trait|
|---|------------------|
|1 | I idolize a particular hero of my faith, and constantly refer to that person's deeds and example.|
|2 | I can find common ground between the fiercest enemies, empathizing with them and always working toward peace.|
|3 | I see omens in every event and action. The gods try to speak to us, we just need to listen.|
|4 | Nothing can shake my optimistic attitude.|
|5 | I quote (or misquote) sacred texts and proverbs in almost every situation.|
|6 | I am tolerant (or intolerant) of other faiths and respect (or condemn) the worship of other gods.|
|7 | I've enjoyed fine food, drink, and high society among my temple's elite. Rough living grates on me.|
|8 | I've spent so long in the temple that I have little practical experience dealing with people in the outside world.|
|d6 | Ideal|
|1 | Tradition. The ancient|

traditions of worship and sacrifice must be preserved and upheld. (Lawful)

|2 | Charity. I always try to help those in need, no matter what the personal cost. (Good)|
|--|--------------------------------------------------------------------------------------|
|3 | Change. We must help bring about the changes the gods are constantly working in the world. (Chaotic)|
```

**Comment by ebullient** [comment]
ooh. good spot. I'll have a look. Thanks!

**Comment by ebullient** [comment]
Which combination of sources are you using? I specifically test against the acolyte background..

**Comment by namtrah** [comment]
I used (as a test) the whole collection CoreRulebooksAndSupplements.xml.

**Comment by ebullient** [comment]
I've found the issue (I think). Will need to disentangle some things.

**Comment by ebullient** [comment]
Resolved by 3bd5cc50a495700ffffa7b5356490e047b4d5df9.

Refresh your binary: 
jbang app install --name fc5-convert --force --fresh https://jitpack.io/dev/ebullient/fc5-convert-cli/1.0.0-SNAPSHOT/fc5-convert-cli-1.0.0-SNAPSHOT-runner.jar

=)
